omnibus/.kitchen.yml

```diff
@@ -2,35 +2,89 @@ driver:
   name: vagrant
   forward_agent: yes
   customize:
-    cpus: 2
-    memory: 2048
+    cpus: 4
+    memory: 4096
   synced_folders:
     - ['..', '/home/vagrant/inspec']
+    - ['../../omnibus', '/home/vagrant/omnibus']
```

Comment:
Unless we have a need to do this, I'd rather keep these out of our default setup.  It is pretty rare to have to build with a custom omnibus/omnibus-software and those who would need to know what to do.  Maybe comment them?